tests/test_migrate_targeted.py

```
@@ -0,0 +1,179 @@
+    """Tests for migrate_gpxtracker function (lines 1048-1081)."""
+
+    def test_returns_success_when_not_configured(self):
+        """Test returns success when gpxtracker not configured."""
+        result = migrate_gpxtracker([], Path("/home"))
+
+        self.assertTrue(result.get("success"))
+        self.assertIn("gpxtracker plugin not configured", result.get("message", ""))
+
+    @patch("yaml.safe_load")
+    @patch("builtins.open")
+    def test_returns_skip_on_yaml_import_error(self, mock_open, mock_yaml_load):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_open`

(ARG002)